README.md

```diff
@@ -113,22 +115,22 @@ E2 - Low power encoding
 
-| Video Processing                             | BDW | SKL | BXT/APL | KBL | CFL | CNL |
-|----------------------------------------------|-----|-----|---------|-----|-----|-----|
-| Blending                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| CSC (Color Space Conversion)                 |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| De-interlace                                 |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| De-noise                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| Luma Key                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| Mirroring                                    |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| ProcAmp (brightness,contrast,hue,saturation) |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| Rotation                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| Scaling                                      |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| Sharpening                                   |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| STD/E (Skin Tone Detect & Enhancement)       |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| TCC (Total Color Control)                    |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| Color fill                                   |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |
-| Chroma Siting                                |  N  |  Y  |    Y    |  Y  |  Y  |  Y  |
+| Video Processing                             | BDW | SKL | BXT/APL | KBL | CFL | CNL | ICL |
+|----------------------------------------------|-----|-----|---------|-----|-----|-----|-----|
+| Blending                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| CSC (Color Space Conversion)                 |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| De-interlace                                 |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| De-noise                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Luma Key                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Mirroring                                    |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| ProcAmp (brightness,contrast,hue,saturation) |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Rotation                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Scaling                                      |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Sharpening                                   |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| STD/E (Skin Tone Detect & Enhancement)       |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| TCC (Total Color Control)                    |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Color fill                                   |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Chroma Siting                                |  N  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
```

Comment:
I thought that as of now we have upstreamed only Chroma Siting in VP, did not we?

Reply:
we up-streamed all of these features for ICL, confirmed with @FurongZhang

Reply:
cool, thank you.